devices/Ft6xx6x/samples/Program.cs

```diff
@@ -1,55 +1,226 @@
+                else if (points == 2)
+                {
+                    var dp = sensor.GetDoublePoints();
+                    Debug.WriteLine($"ID: {dp.Point1.TouchId}, X: {dp.Point1.X}, Y: {dp.Point1.Y}, Weight: {dp.Point1.Weigth}, Misc: {dp.Point1.Miscelaneous}, Event: {dp.Point1.Event}");
+                    Debug.WriteLine($"ID: {dp.Point2.TouchId}, X: {dp.Point2.X}, Y: {dp.Point2.Y}, Weight: {dp.Point2.Weigth}, Misc: {dp.Point2.Miscelaneous}, Event: {dp.Point1.Event}");
+                }
+            }
+
+            void DiscrimitateButtons(int x, int y)
+            {
+                const double pxPerMm = 6.90415;
+                //r in milimeters is the radious of sensitivity to detect any of the given 3 buttons
+                const double rMm = 3.5;
+                const int x0L = 83;
+                const int x0M = 182;
+                const int x0R = 271;
+                const int y0 = 263;
+
+                TouchEventArgs touchEventArgs = new TouchEventArgs();
+
+                //WARNING: Next assigment should be EventCategory.Touch but Touch Category not yet defined on main/develop
+                touchEventArgs.eventCategory = EventCategory.Unknown;
+                touchEventArgs.X1 = x;
+                touchEventArgs.Y1 = y;
+
+                //r^2 in pixels^2
+                int r2 = (int)((rMm * pxPerMm) * (rMm * pxPerMm));
+
+                int tmpL = 0;
+                int tmpM = 0;
+                int tmpR = 0;
+                int tmpY = 0;
+
+                tmpY = (y - y0) * (y - y0);
+                tmpL = (x - x0L) * (x - x0L);
+                tmpL += tmpY;
+                tmpM = (x - x0M) * (x - x0M);
+                tmpM += tmpY;
+                tmpR = (x - x0R) * (x - x0R);
+                tmpR += tmpY;
+
+                if (tmpL <= r2)
+                {
+                    touchEventArgs.eventSubCategory = (int)TouchEventSubcategory.LeftButton;
+                }
+                else if (tmpM <= r2)
+                {
+                    touchEventArgs.eventSubCategory = (int) TouchEventSubcategory.MiddleButton;
+                }
+                else if (tmpR <= r2)
+                {
+                    touchEventArgs.eventSubCategory = (int) TouchEventSubcategory.RightButton;
+                }
+                else
+                {
+                    touchEventArgs.eventSubCategory = (int) TouchEventSubcategory.SingleTouch;
+                }
+
+                TouchPanelProcessor.OnTouchedEvent(touchEventArgs.eventSubCategory, touchEventArgs.X1, touchEventArgs.Y1);
+
+            }
+
+            // Browse our samples repository: https://github.com/nanoframework/samples
+            // Check our documentation online: https://docs.nanoframework.net/
+            // Join our lively Discord community: https://discord.gg/gCyBu8T
+        }
+
+    }
+    public class TouchEventArgs : EventArgs
+    {
+        public EventCategory eventCategory { get; set; }
```

Comment:
Public properties and fields names start with upper case please.

Reply:
sorry, in that specific case the property would have the same name as the class...i changed that and Capitalized the first letter..(DONE ...on next PR)